.github/workflows/main.yml

```diff
@@ -1,17 +1,32 @@
-name: CI
+name: meetup-talks
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v1
-    - name: Run a one-line script
-      run: echo Hello, world!
-    - name: Run a multi-line script
-      run: |
-        echo Add other actions to build,
-        echo test, and deploy your project.
+      - name: Setup linkstatus, mdspell and write-good
+        run: |
+          python3 -m pip install linkstatus --user
+          sudo apt install nodejs
+          sudo npm i markdown-spellcheck -g
+          sudo npm install write-good -g
+
+      - name: Checkout to changes
+        uses: actions/checkout@v1
+
+      - name: Check Links
+        run: |
+          changes=`git diff remotes/origin/${{ github.base_ref }} --name-only`
```

Comment:
I think we should exclude the deleted files from this. 
`git diff --diff-filter=d` will do the trick I believe. 
https://stackoverflow.com/a/41730200/6202405

Reply:
yup... lets ignore deleted files